Amplify/Categories/Analytics/AnalyticsCategory+ClientBehavior.swift

```diff
@@ -6,8 +6,8 @@
 //
 
 extension AnalyticsCategory: AnalyticsCategoryBehavior {
-    public func identifyUser(_ identityId: String, withProfile userProfile: AnalyticsUserProfile? = nil) {
-        plugin.identifyUser(identityId, withProfile: userProfile)
+    public func identifyUser(userId: String, userProfile: AnalyticsUserProfile? = nil) {
+        plugin.identifyUser(userId: userId, userProfile: userProfile)
```

Comment:
is it intentional that the api requires an argument label now? or should it have an optional argument label i.e. `_` ?